internal/logging/zerolog.go

```diff
@@ -62,8 +61,11 @@ func InitializeStdout() {
 		panic(fmt.Errorf("cannot parse log level '%s': %w", config.Logging.Level, err))
 	}
 	zerolog.SetGlobalLevel(level)
+
 	log.Logger = decorate(log.Output(zerolog.ConsoleWriter{
-		Out:        os.Stdout,
+		Out: os.Stdout,
+		// Do not use colors when running in clowder
+		NoColor:    config.InClowder(),
```

Comment:
Are we sure we want to disable colors in stage env? the openshift logging shows the colors pretty well and it improves readability IMHO

Reply:
There is no terminal you can connect to, therefore it shows up unreadable. See the example here: https://github.com/RHEnVision/provisioning-backend/pull/255#issuecomment-1285484097

Reply:
Well, there is none in the tests, but there is one in the stage environment namespace.
So I'd love to distinguish those two environments.

Reply:
It`s now info by default.